### ISSUE TITLE
new release

### ISSUE THREAD

**Issue by Conni2461** [NuML/NuML#30]
the last release is quite some time ago and it would be great if there is a new release with python3.11, 3.12 and swig4.1 support.

Is there any chance that a new release/tag can be published? I am currently working on packaging libnuml for [nixpkgs](https://github.com/nixOS/nixpkgs) and using the latest release rather than the latest commit would be rather nice :)

**Comment by fbergmann** [comment]
currently on a conference this week, I can try and work on this next week tho

**Comment by Conni2461** [comment]
Sounds great, thank you very much :)

**Comment by luciansmith** [comment]
We also are starting to have users that would like to use python 3.13; it would be nice to get a 3.13 release.

**Comment by fbergmann** [comment]
yes it definitely is time for a new release. thanks for pinging me.

**Comment by matthiaskoenig** [comment]
Same here, want to use python 3.13

**Comment by luciansmith** [comment]
Thank you!
